1. Починил все тесты, привел код в порядок

Dmitriy-Evs opened the pull request:
2. Добавил отчет аллур

Dmitriy-Evs committed:
1. Починил все тесты, привел код в порядок
2. Добавил отчет аллур

egorvas commented on src/test/java/CreateCourierTest.java:
```diff
@@ -0,0 +1,88 @@
+    static private String PEN_DELETE = "/api/v1/courier/";
+    @Before
+    public void setUp() {
+        RestAssured.baseURI = "http://qa-scooter.praktikum-services.ru/";
```
**Можно лучше:**baseURI лучше выносить в одно общее место для всех тестов. На практике твои тесты будут запускаться на разных энвах: дев, прод, тест и тп и везде будут разные url. По этому чтобы в каждом тесте не менять baseURI лучше это делать один раз для всех тестов.

egorvas commented on src/test/java/CreateCourierTest.java:
```diff
@@ -0,0 +1,88 @@
+    @DisplayName("Создание нового курьера с правильными данными")
+    public void createNewCourierAndCheckResponse(){
+        Courier courierCreate  = new Courier(login, password, firstName);
+        given()
```
**Можно лучше:** я крайне рекомендую тебе вынести все запросы в отдельные функции - Степы. Это позволит упростить твои тесты, сделает их более читабельными, даст возможность переиспользовать запросы. Также если ты добавишь аннотацию @Step, то твои тесты будут отображаться красиво в отчете allure.

egorvas commented on src/test/java/LoginCourierTest.java:
```diff
@@ -0,0 +1,154 @@
+
+public class LoginCourierTest {
+    static private String login = RandomStringUtils.randomAlphabetic(10);
+    static private String passwordTest = "11112222";
```
**Можно лучше:** Вместо того чтобы самому придумывать тестовые строки я рекомендую использовать модули для генерации данных, например java faker: https://github.com/DiUS/java-faker
С его помощью ты убрать большую часть хардкора из твоих тестов.

Dmitriy-Evs committed:
1. Выделил адрес стенда в отдельный класс, теперь можно поменять адрес в одном месте.
2. Использовал тот же модуль для генерации данных, что уже был в тесте.
3. Для теста создания курьеров создал шаги для отправки запросов, выделил их в отдельный класс. По совету наставника оставил сами проверки в рамках теста, не создавал для них шагов.
